### PR TITLE
Create Milestone Delivery infinitywallet_milestone_1+2.md

### PR DESCRIPTION
**Pull request by InfinityWallet**
# Grant Application Checklist

- [X] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [X] An Ethereum address for the payment of the milestones is provided inside the application.  
- [X] The total funding amount of the project is up to USD $30k for every grant applications.
- [X] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

**Review by libotony — CHANGES_REQUESTED** [review]
Could you kindly separate it into two files, named xxx-milestone_1.md and xxx-milestone_1.md? And also attach the video link you have posted before.

**Comment by InfinityWallet** [comment]
> Could you kindly separate it into two files, named xxx-milestone_1.md and xxx-milestone_1.md? And also attach the video link you have posted before.

We have split the milestones into 2 files and attached the video link to milestone 2 file

File: milestone-delivery/deliveries/infinitywallet_milestone_1.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Milestone Delivery :mailbox:
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/186
+* **Milestone Number:** Milestone 1
```
Comment (libotony) [inline milestone-delivery/deliveries/infinitywallet_milestone_1.md:4]:
```suggestion
* **Milestone Number:** 1
```

File: milestone-delivery/deliveries/infinitywallet_milestone_2.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Milestone Delivery :mailbox:
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/186
+* **Milestone Number:** Milestone 1 2
```
Comment (libotony) [inline milestone-delivery/deliveries/infinitywallet_milestone_2.md:4]:
```suggestion
* **Milestone Number:** 2
```

**Review by libotony — CHANGES_REQUESTED** [review]

**Review by arons1 — APPROVED** [review]

**Review by libotony — APPROVED** [review]

**Review by NecoSherry — APPROVED** [review]

**Review by Shazeli99 — COMMENTED** [review]
How to use withdrawal accepted gives away

**Pull request merged**